tests/test_refObjLoader.py

```diff
@@ -0,0 +1,294 @@
+        # on cos(dec) and the tangent-plane projection.
+        self.assertLess(np.max(np.rad2deg(cat['coord_ra'])), 180.0 + 25.0)
+        self.assertGreater(np.max(np.rad2deg(cat['coord_ra'])), 180.0 - 25.0)
+        self.assertLess(np.max(np.rad2deg(cat['coord_dec'])), 25.0)
+        self.assertGreater(np.min(np.rad2deg(cat['coord_dec'])), -25.0)
+
+        # The following is to ensure the reference catalog coords are
+        # getting corrected for proper motion when an epoch is provided.
+        # Use an extreme epoch so that differences in corrected coords
+        # will be significant.  Note that this simply tests that the coords
+        # do indeed change when the epoch is passed.  It makes no attempt
+        # at assessing the correctness of the change.  This is left to the
+        # explicit testProperMotion() test below.
+        catWithEpoch = loader.loadPixelBox(
+            bbox,
+            wcs,
+            'a',
+            bboxToSpherePadding=0,
+            epoch=astropy.time.Time(30000, format='mjd', scale='tai')).refCat
+
+        self.assertFloatsNotEqual(cat['coord_ra'], catWithEpoch['coord_ra'], rtol=1.0e-4)
+        self.assertFloatsNotEqual(cat['coord_dec'], catWithEpoch['coord_dec'], rtol=1.0e-4)
+
+    def test_filterMap(self):
+        """Test filterMap parameters."""
+        loaderConfig = ReferenceObjectLoader.ConfigClass()
+        loaderConfig.filterMap = {'aprime': 'a'}
+        loader = ReferenceObjectLoader([dataRef.dataId for dataRef in self.datasetRefs],
+                                       self.handles,
+                                       config=loaderConfig)
+        center = lsst.geom.SpherePoint(180.0*lsst.geom.degrees, 0.0*lsst.geom.degrees)
+        result = loader.loadSkyCircle(
+            center,
+            30.0*lsst.geom.degrees,
+            filterName='aprime',
+        )
+        self.assertEqual(result.fluxField, 'aprime_camFlux')
+        self.assertFloatsEqual(result.refCat['aprime_camFlux'], result.refCat['a_flux'])
+
+    def test_properMotion(self):
+        """Test proper motion correction."""
+        loaderConfig = ReferenceObjectLoader.ConfigClass()
+        loaderConfig.filterMap = {'aprime': 'a'}
+        loader = ReferenceObjectLoader([dataRef.dataId for dataRef in self.datasetRefs],
+                                       self.handles,
+                                       config=loaderConfig)
+        center = lsst.geom.SpherePoint(180.0*lsst.geom.degrees, 0.0*lsst.geom.degrees)
+        cat = loader.loadSkyCircle(
+            center,
+            30.0*lsst.geom.degrees,
+            filterName='a'
+        ).refCat
+
+        # Zero epoch change --> no proper motion correction (except minor numerical effects)
+        cat_pm = loader.loadSkyCircle(
+            center,
+            30.0*lsst.geom.degrees,
+            filterName='a'
+        ).refCat
+        loader.applyProperMotions(cat_pm, self.epoch)
```

Comment:
This is unnecessary if you pass `epoch` to `loadSkyCircle`, and that's the way it is used in practice.